docs/assert/rejects.md

```diff
@@ -26,6 +26,7 @@ comparison.
 The `expectedMatcher` argument can be:
 
 * A function that returns `true` when the assertion should be considered passing.
+* An Error Object (instance)
```

Comment:
I think to avoid potential confusion, it might be better to lowercase `Object` so nobody thinks it might refer to the global Object constructor. And I'm not sure the parentheses are necessary either.

So something like:

> * An Error object instance

Or:

> * An Error object

What do you think?

Reply:
Good point.

I will update it to `An Error object` as it also include plain objects with `name` and `message` properties (those are still used by some developers out there, e.g. `{name: "AuthenticationError", message: "Wrong username or password"}`)